Rollbar/Rollbar.h

```diff
@@ -80,6 +80,10 @@
 
 + (void)logCrashReport:(NSString*)crashReport;
 
+// Send JSON payload
+
++ (void)send:(NSData*)payload;
```

Comment:
I thought about that. The tradeoff is `send` is more consistent with the naming that is (now) introduced across the other SDK interfaces. The internal notifier naming differentiates `sendItem` and `sendPayload`. I suppose we could push that internal detail up to the public interface here. I was inclined to prefer the simpler `send` as long as it wouldn't be ambiguous in any context.

Reply:
I should add that in some SDKs (Java), `Payload` is a type and therefore `sendPayload` infers more that we would want. To the extent we want this to be consistent, like we do with `log()`, that's a reason to avoid `sendPayload` in the public interface.

Reply:
we should be consistent across the SDKs semantically as much as possible, but necessarily syntactically. for example, in .NET/C# all the public methods are capitalized by convention and not all languages support method overloads (objective-c is somewhat different here: [https://learning.oreilly.com/library/view/ios-6-programming/9781449342746/ch01s15.html](https://learning.oreilly.com/library/view/ios-6-programming/9781449342746/ch01s15.html) )

Reply:
Good point.

Reply:
Cocoa guidelines (https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html) say sendPayload is probably the preferred name, although I hate quite a few of their guidelines.